Packs/RecordedFuture/IncidentTypes/customIncidentTypes.json

```diff
@@ -0,0 +1,42 @@
+[
+	{
+		"id": "RecordedFuture",
```

Comment:
I think we should think of a better name here
@evisochek What do you think?

Reply:
I thinks its worth to add couple of words for better description of the incident type

Reply:
Change the file name
@rderkachrf